src/components/footer/www/footer.jsx

```diff
@@ -81,17 +81,17 @@ var Footer = React.createClass({
                                 </a>
                             </dd>
                             <dd>
-                                <a href="/info/credits/">
+                                <a href="/info/credits">
                                     <FormattedMessage id='general.credits' />
                                 </a>
                             </dd>
                             <dd>
-                                <a href="/jobs/">
+                                <a href="/jobs">
                                     <FormattedMessage id='general.jobs' />
                                 </a>
                             </dd>
                             <dd>
-                                <a href="http://wiki.scratch.mit.edu/wiki/Scratch_Press">
+                                <a href="http://wiki.scratch.mit.edu/wiki/Scratch_Press/">
```

Comment:
Since this is an external link to the wiki, I think we should leave it as is (without the trailing slash)